src/tests/Microsoft.PowerFx.Interpreter.Tests/CollectFunctionTests.cs

```diff
@@ -1,155 +0,0 @@
-﻿// Copyright (c) Microsoft Corporation.
-// Licensed under the MIT license.
-
-using System;
-using System.Collections.Generic;
-using System.Threading;
-using System.Threading.Tasks;
-using Microsoft.PowerFx.Core.App.ErrorContainers;
-using Microsoft.PowerFx.Core.Binding;
-using Microsoft.PowerFx.Core.Errors;
-using Microsoft.PowerFx.Core.Functions;
-using Microsoft.PowerFx.Core.Functions.FunctionArgValidators;
-using Microsoft.PowerFx.Core.Localization;
-using Microsoft.PowerFx.Core.Tests;
-using Microsoft.PowerFx.Core.Types;
-using Microsoft.PowerFx.Core.Utils;
-using Microsoft.PowerFx.Interpreter;
-using Microsoft.PowerFx.Syntax;
-using Microsoft.PowerFx.Types;
-using Xunit;
-using static Microsoft.PowerFx.Core.Localization.TexlStrings;
-
-namespace Microsoft.PowerFx.Interpreter.Tests
-{
-    public class CollectFunctionTests : PowerFxTest
```

Comment:
>CollectFunctionTests [](http://example.com/codeflow?start=17&length=20)

deleted?

Reply:
These test cases have been moved to Collect.txt

Reply:
Moved back some SDK test cases.